ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity.component.ts

```diff
@@ -91,9 +91,7 @@ export class ActivityComponent extends BaseActivityComponent implements OnInit,
     public ngOnInit(): void {
         this.getActivity();
         this.initStepperState();
-        const submitSub = this.submitAttempted.subscribe((isSubmit: boolean) => {
-            this.submitService.announceSubmit(isSubmit);
-        });
+        const submitSub = this.submitAttempted.subscribe(() => this.submitAnnouncementService.announceSubmit(null));
```

Comment:
are we sure we want to ignore the subject value and always pass null?

Reply:
We should probably convert the type to void. It is just a signal that submit is occurring. And sometime we don't have the body anyways.

Reply:
c41ed31263e59cf692d6c16b0cb2b283ba41555a